train.py

```diff
@@ -198,13 +200,16 @@ def train(self):
 
     best_valid_loss = 1.e6
     for epoch in range(self.startEpoch, self.params.max_epochs):
+      logging.info("Epoch: %d" % epoch)
```

Comment:
```suggestion
      logging.info("Epoch: %d" % epoch + 1)
```

1-based index probably makes more sense here, and is already the precedent set on line 241.